design-principles.md

```diff
@@ -0,0 +1,83 @@
+
+## 🤔 How can you help?
+
+**Share your feedback on this document!** One of our proposed principles is “**Collaborative Improvement**” which means that we want the website to be the product of many contributors. So in the spirit of that principle, we want to share these design principles with the Ethereum community.
+
+While these principles are focused on the [ethereum.org](http://ethereum.org) website, we hope that many of them are representative of the values of the Ethereum ecosystem overall (e.g. you can see influence from the [principles of the Ethereum Whitepaper](https://github.com/ethereum/wiki/wiki/White-Paper#philosophy)). Maybe you even want to incorporate some of them into your own project!
+
+---
+
+## Design Principles for Ethereum.org
+
+## 1. Inspirational
+
+The site should inspire users to dream of how Ethereum can change the world. It should motivate people to explore, play and tinker with the tools and apps of the Ethereum ecosystem.
+
+- **Radical:** The site should communicate Ethereum's ambitious goals to meaningfully change the world. It should be clear that Ethereum is not just some new product - it is a revolutionary technology.
+- **Empowerment through education:** The site should educate people so they can understand Ethereum's potential, find their place in the ecosystem, and feel empowered to participate in it.
+
+Visual Direction • Content
+
+## 2. Universal
+
+Ethereum is a global, decentralized project and our audience reflects this. The site should aspire to be accessible to everyone, and sensitive to the world’s many cultures.
+
+- **Accessible:** The site should follow accessibility guidelines - including for people with low-bandwidth connections.
+- **Straightforward:** The site should be simple and unambiguous. Copy shouldn’t use language that may be misinterpreted or lost in translation.
+- **Ethereum is multi-faceted:** Ethereum is a project, a codebase, a community, and a vision. Ethereum is valuable to different people for different reasons, and there are many ways to be involved.
+
+Writing systems • Use of colour • Visual Direction • Content
+
+## 3. A Good Story
+
+The website should function like a good story. Visitors are on a journey, and the content you contribute is a part of that. Your contributions should fit within a clear narrative: one with a beginning (introduction/entry point), middle (set of learnings and insights), and end (a link(s) to relevant resources, or next steps).
+
+- **Hierarchical**: A clear, hierarchically structured information architecture helps visitors to [ethereum.org](http://ethereum.org) navigate the site "as a story" as they seek to achieve their goals.
+- **A Stepping Stone:** We don’t want to replace or become a substitute for the incredible resources that already exist. We’re a stepping stone - people come looking for an answer. We give an answer & provide trustworthy next steps.
```

Comment:
> **A Stepping Stone:** We don’t want to replace or become a substitute for the incredible resources that already exist. We’re a stepping stone - people come looking for an answer. We give an answer & provide trustworthy next steps.

**A Stepping Stone:** We’re a stepping stone for people come looking for an answer. We don’t want to replace or become a substitute for the many resources that already exist. We give an answer & provide reliable next steps.

Personally, I think this order works better. Also think some words like "trustworthy" should be used sparingly if at all in the context of eth.

Reply:
agree!